src/ZString.Unity/Assets/Scripts/ZString/FormatHelper.cs

```diff
@@ -48,17 +48,17 @@ private static void FormatToRightJustify<TBufferWriter, T>(ref TBufferWriter sb,
             if (typeof(T) == typeof(string))
             {
                 var s = Unsafe.As<string>(arg);
-                int padding = width - s.Length;
+                int padding = width - s!.Length;
```

Comment:
The change is correct, but should be applied at the point where `s` is initialized, like this - `var s = Unsafe.As<string>(arg)!;`. That way there would be no need to modify all the usages of `s`.